src/overlays/actors/ovl_En_Ta/z_en_ta.c

```diff
@@ -519,28 +519,28 @@ s32 func_80B14DD8(void) {
 }
 
 void func_80B14E28(EnTa* this, GlobalContext* globalCtx) {
-    Vec3f b;
-    Vec3f a;
+    Vec3f subCamEye;
+    Vec3f subCamAt;
 
-    this->unk_2D0 = Gameplay_CreateSubCamera(globalCtx);
-    this->unk_2D2 = globalCtx->activeCamera;
-    Gameplay_ChangeCameraStatus(globalCtx, this->unk_2D2, CAM_STAT_WAIT);
-    Gameplay_ChangeCameraStatus(globalCtx, this->unk_2D0, CAM_STAT_ACTIVE);
+    this->subCamId = Gameplay_CreateSubCamera(globalCtx);
+    this->activeCamId = globalCtx->activeCamId;
+    Gameplay_ChangeCameraStatus(globalCtx, this->activeCamId, CAM_STAT_WAIT);
```

Comment:
another spot where "active" is confusing, that's the previously active camera

Reply:
Would `this->returnToCamId` work in this case? This would apply to `en_ta`, `en_kz`, and `en_in` as they all have this same pattern and current name

Reply:
Sure! can discuss it further in the other thread, but it's ok to me. as long as the name isn't misleading (like `returnToCamId` is not) and is consistently used